packages/driver/src/cypress/cy.ts

```diff
@@ -896,6 +899,10 @@ export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssert
   }
 
   now (name, ...args) {
+    if (this.queryFns[name]) {
+      return this.queryFns[name].apply(this.state('current'), args)
```

Comment:
Shouldn't the first argument be `this` so the context is shared for query commands? And the second subject would be `this.state('current')`?